src/main/webapp/query-control.js

```diff
@@ -56,23 +57,27 @@ function passQuery() {
   } else {
     title += ' in ';
   }
-  title += 'each ' + region + ' (' + personType.replace('-', ' ') + ')';
+  title += 'each ' + region + ' (' +
+      personType.replace('-', ' ') + ')' +
+      ' in ' + year;
   document.getElementById('map-title').innerText = title;
 
   const fetchUrl = '/query?person-type=' + personType +
     '&action=' + action +
-    '&location=' + location;
+    '&location=' + location +
+    '&year=' + year;
 
   fetch(fetchUrl)
     .then((response) => {
       if (response.ok) {
-        response.json().then((jsonResponse) => JSON.parse(jsonResponse))
-          .then((censusDataArray) => {
+        response.json()
+        .then((response) => {
             // censusDataArray is a 2D array, where the first row is a
             // header row and all subsequent rows are one piece of
             // data (e.g. for a state or county)
-            displayVisualization(censusDataArray, description,
+            displayVisualization(JSON.parse(response.data), description,
               location, isCountyQuery);
+            displayLinkToCensusTable(response.table, year);
```

Comment:
I like the way you set this up, so that the info goes from the backend, to the control, and then directly to a new JS function (rather than getting dragged along through all our other JS functions).